### PR TITLE
aa/MTV 428 document insecure connections to rhv

### PR DESCRIPTION
**Pull request by anarnold97**
@ahadas @bkhizgiy @qiyuann

Apologies, I have had to create a new [MTV-428](https://issues.redhat.com/browse/MTV-428) PR for this as the earlier PR was a mess

See [Mig virt mach cli hvider #373 ](https://github.com/kubev2v/forklift-documentation/pull/373) for your earlier comments that I have included in this PR.

Please, could you review and do the necessary? 

Thanks

### CODE REVIEW

**Review by qiyuann — APPROVED** [review]
LGTM

**Comment by anarnold97** [comment]
Thanks @qiyuann, appreciate your help

**Review by bkhizgiy — APPROVED** [review]
Thanks @anarnold97, looking good, approved form my side.

**Review by ahadas — APPROVED** [review]

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -59,26 +59,20 @@ metadata:
...
-  cacert: | <8>
-    <ca_certificate>
-  url: <api_end_point> <9>
-  thumbprint: <vcenter_fingerprint> <10>
+
+  user: <user> <1>
+  password: <password> <2>
+  insecureSkipVerify: <true/false> <3>
+  cacert: | <4>
+    <engine_ca_certificate>
```
Comment (ahadas) [inline documentation/modules/migrating-virtual-machines-cli.adoc:67]: we've changed the description of the cacert field - it is also relevant for openstack now, so changing the ca_certification to engine_ca_certificate is incorrect

**Review by ahadas — COMMENTED** [review]
need to rebase this PR to get the change I mentioned in line and other changes that were made for openstack

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 2]
```diff
@@ -59,26 +59,20 @@ metadata:
...
-<6> {osp} only: Specify the project name.
-<7> {osp} only: Specify the name of the {osp} region.
-<8> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment.
-<9> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api/` for {rhv-short}, or `https://<identity_service>/v3` for {osp}.
-<10> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> Specify the vCenter user or the {rhv-short} {manager} user.
+<2> Specify the user password.
+<3> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and that the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
```
Comment (ahadas) [inline documentation/modules/migrating-virtual-machines-cli.adoc:73]: not sure about this but should "that the certificate" change to "then the certificate"?
```suggestion
<3> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
```

File: documentation/modules/migrating-virtual-machines-cli.adoc (hunk 1 above)
Comment (anarnold97) [reply documentation/modules/migrating-virtual-machines-cli.adoc:67]: @ahadas please can you clarify?
Thanks

Comment (ahadas) [reply documentation/modules/migrating-virtual-machines-cli.adoc:67]: the `cacert` field is also used now to set the certificate of an openstack provider so it's incorrect to rename it to `engine_ca_certificate` and change it back to `{rhv-short} only`

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 3]
```diff
@@ -59,26 +59,20 @@ metadata:
...
-<3> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
-<4> Specify the user password.
-<5> {osp} only: Specify the domain name.
-<6> {osp} only: Specify the project name.
-<7> {osp} only: Specify the name of the {osp} region.
-<8> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment.
-<9> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api/` for {rhv-short}, or `https://<identity_service>/v3` for {osp}.
-<10> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> Specify the vCenter user or the {rhv-short} {manager} user.
```
Comment (ahadas) [inline documentation/modules/migrating-virtual-machines-cli.adoc:71]: several fields that were added for openstack were dropped during the rebase

Comment (anarnold97) [reply documentation/modules/migrating-virtual-machines-cli.adoc:71]: @ahadas - we good to merge, please?

**Review by ahadas — APPROVED** [review]
I rebased the PR in a way that keeps the additions for OpenStack , @anarnold97 please double check me

**Pull request merged**